features/models.py

```diff
@@ -67,6 +68,12 @@ class Feature(TranslatableModel, TimestampedModel):
         url=models.URLField(
             verbose_name=_("url"), blank=True, help_text=_("URL of the feature")
         ),
+        one_liner=models.CharField(
+            verbose_name=_("one-liner"),
+            max_length=50,
```

Comment:
Do I remember wrong, or was the limit initially 250? It's just that 50 sounds awfully short... I don't think we should be too strict about it in the backend. Looking at the designs, I think we might easily have 60-70 characters.

Reply:
I leave it to 64 and create a ticket in Jira that we double check with the FE guys when the element is implement in FE and we get some justifiable value.